Backend-tests.md

```diff
@@ -151,33 +151,57 @@ In fact, too many mocks can be a problem because when someone changes the code y
 
 You can run backend tests like this:
 
+Python:
 ```console
 python -m scripts.run_backend_tests
 ```
 
+Docker:
+```console
+make run_tests.backend
+```
+
 Alternatively, you can run just a single test module like this:
 
+Python:
 ```console
 python -m scripts.run_backend_tests --test_target=core.controllers.editor_test
 ```
 
+Docker:
+```console
+make run_tests.backend PYTHON_ARGS="--test_target=core.controllers.editor_test"
```

Comment:
this is also throwing 
![Screenshot from 2024-02-08 18-11-21](https://github.com/oppia/oppia-web-developer-docs/assets/127002519/0c2be13c-cc82-4ebb-94a3-c738ca65fe10)